react/modules/checkoutHook.ts

```diff
@@ -3,13 +3,15 @@ import { useRuntime } from 'vtex.render-runtime'
 
 export default function useCheckout() {
   const { url: checkoutUrl, major } = useCheckoutURL()
-  const { navigate } = useRuntime()
+  const { navigate, rootPath = '' } = useRuntime()
 
   const goToCheckout = (url: string) => {
+    const urlWithRootPath = `${rootPath}${url}`
+
     if (major > 0 && url === checkoutUrl) {
-      navigate({ to: url })
+      navigate({ to: urlWithRootPath })
```

Comment:
I don't think we need the root path here, [`render-runtime` already adds it by default](https://github.com/vtex-apps/render-runtime/blob/f9b1d41333da341c0e27604f1f747a9ec953c45a/react/utils/pages.ts#L363-L370).

Reply:
Updated.